packages/expect/src/__tests__/isError.test.ts

```diff
@@ -34,11 +36,11 @@ describe('isError', () => {
   });
 
   it('should detect errors from another context', () => {
-    testErrorFromDifferentContext(win => new win.Error());
+    testErrorFromDifferentContext((win: any) => new win.Error());
```

Comment:
`Window` or `typeof window` or something could be a good type here?

Reply:
`Window` (as defined in lib.dom.ts) does not have `Error` property:
`TS2339: Property 'Error' does not exist on type 'Window'.`
We could extend the `Window` declaration, or use the `NodeJS.Global` type. Any preferences?

Reply:
That is weird. It works on the playground [Playground Link](https://www.typescriptlang.org/play?#code/HYUw7gBGCWwCYHswDoCiAndD0AoCUQA)

Reply:
You are right. The TS error only shows up in my IDE... so i guess my IDE loads the TS settings badly :-/